ntbs-service/Pages/Migration/Index.cshtml.cs

```diff
@@ -19,12 +25,21 @@ public IndexModel(INotificationImportService service)
         {
             _service = service;
             Results = new List<Notification>();
+            ValidationService = new ValidationService(this);
         }
 
         [BindProperty]
         public string NotificationId { get; set; }
 
+        [BindProperty]
+        public IFormFile UploadedFile { get; set; }

```

Comment:
Is there any validation that should be applied to the file? Max size etc.? Not sure what might be available out of the box